test/test.js

```diff
@@ -397,6 +397,12 @@ describe('vite', () => {
         moment(1590231082886).format('MMMM Do YYYY, h:mm:ss a')
       )
     })
+
+    if (!isBuild) {
```

Comment:
To support `build`, we need to release `rollup-plugin-vue@ 6.0.0-beta.4`.
https://github.com/vuejs/rollup-plugin-vue/commit/0720102dd5d023a19bb720329cc84d91c5d88f7c